### ISSUE TITLE
Automatic squeeze of a 3D Array

### ISSUE THREAD

**Issue by arsenal9971** [JuliaComputing/ArrayFire.jl#146]
When I am trying to create an ArrayFire of size (n,m,1), it automatically creates an array of size (n,m), namely

```julia
using ArrayFire
A1 = AFArray(zeros(Float32,3,3,1));
A2 = zeros(Float32,3,3,1);
```
the size of `A1` is (3,3) wheres the size of `A2` is (3,3,1)

**Comment by ranjanan** [comment]
@arsenal9971 You are right, the underlying library reasons about sizes that way. It is definitely a deviation from how Julia behaves (Ref #75 )

**Comment by gaika** [comment]
fixed in #121

**Comment by arsenal9971** [comment]
Thanks!!
